masterlist.yaml

```diff
@@ -613,6 +621,26 @@ plugins:
       - 'IWR.esm'
 #  - name: 'RBX.esm'
 # Needs to be before any mod using a reloading bench including a lot of house mods.
+  - name: 'RobCo Certified.esm'
+    dirty:
+      - <<: *quickClean
+        crc: 0x3B28B2FE
+        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 9
+    reqManualFix:
```

Comment:
This won't do anything, it's not recognised syntax.

Reply:
How would you fix this? just add a msg?

Reply:
Since there are only two affected records, I'd just include them in the dirty info:

```
    dirty:
      - <<: *quickClean
        crc: 0x3B28B2FE
        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
        itm: 9
      - crc: 0xAA3DB25C
        util: '[FNVEdit v4.0.1i EXPERIMENTAL](https://www.nexusmods.com/newvegas/mods/34703)'
        info: 'Contains wild edits to records 0x00094684 (DLC01TrogFaction) and 0x00137622 (FortCaesarScript).'
```

Reply:
I'll fix that and make a new pull request. Thanks